tests/test_vpc.rb

```diff
@@ -379,11 +378,5 @@ def test_interface_vpc_peer_link
     refute(interface_pc.vpc_peer_link, 'vpc_peer_link should not be set')
     # clean up
     interface.channel_group = false
-    #
-    # negative - try on a phy port
```

Comment:
Yeah since the interface_channel_group object cannot be used to apply interface properties. Earlier the property was applied on a interface.